spec/spacex/launches_spec.rb

```diff
@@ -393,6 +393,27 @@
     end
   end
 
+  context '#upcoming', vcr: { cassette_name: 'launches/upcoming' } do
+    subject do
+      SPACEX::Launches.upcoming
+    end
+
+    it 'returns and array of upcoming launch hashes' do
+      expect(subject).to be_an Array
+      expect(subject.first).to be_a Hash
+      expect(subject.all?(&:upcoming)).to be true
+    end
+
+    it 'returns launches scheduled in the future' do
```

Comment:
I like these tests. Can you also add one that tests to make sure we get all the expected fields for each launch? Specs for other end points are doing it and it helps us if ever the actual endpoint changes the data being returned (when we rerecord the cassette of course)

Reply:
Alright added some tests to check all of the fields. Let me know if that's good.